Dockerfile

```diff
@@ -273,21 +273,23 @@ USER root
 
 RUN curl -o- -L https://yarnpkg.com/install.sh > /usr/local/bin/yarn-installer.sh
 
+ENV NVM_VERSION=0.35.3
+
 # Install node.js
 USER buildbot
 RUN git clone https://github.com/creationix/nvm.git ~/.nvm && \
     cd ~/.nvm && \
-    git checkout v0.34.0 && \
+    git checkout v$NVM_VERSION && \
     cd /
 
 ENV ELM_VERSION=0.19.0-bugfix6
 ENV YARN_VERSION=1.13.0
 
-ENV NETLIFY_BUILD_NODE_VERSION="12.16.1"
+ENV NETLIFY_NODE_VERSION="10.20.1"
```

Comment:
Why is this `NETLIFY_NODE_VERSION` and not just `NODE_VERSION`?

Reply:
No particular reason, more that it represents the version Netlify has chosen for the image. :shrug:

Reply:
Note that `NODE_VERSION` has a specific meaning, since it's the name of the environment variable used by users to specify a custom Node.js version (as both of you know). I am not sure whether that would be a good thing or not in this instance.

Reply:
Yeah, that shouldn't interfere. These ENV's are only present during the docker build phase.

Reply:
Ok, that makes sense.